### PR TITLE
Feature/bookmarks

### PR DESCRIPTION
**Pull request by gjeanmart**

### CODE REVIEW

**Comment by davodesign84** [comment]
100 files PR :scream:

File: app/queries/User.ts [hunk 1]
```diff
@@ -120,3 +121,86 @@ export const emailSubscribe = gql`
         }
     }
 `
+
+export const getBookmarkFolders = gql`
```
Comment (davodesign84) [inline app/queries/User.ts]: Is it worth having a separate Bookmark file for the queries?

**Comment by gjeanmart** [comment]
Fixed all the comments and the conflicts with master!

File: app/queries/User.ts (hunk 1 above)
Comment (gjeanmart) [reply app/queries/User.ts]: done. see `queries/Bookmark.ts`

File: app/lib/getURLs.ts [hunk 2]
```diff
@@ -62,6 +62,11 @@ export const getProfileURL = ({ username }: IProfileLinkProps) => ({
     href: `/public-profile?username=${username}`,
 })
 
+export const getBookmarksURL = () => ({
+    as: `/bookmarks`,
```
[the remark this answers is not in the document]
Comment (gjeanmart) [reply app/lib/getURLs.ts]: done

File: app/containers/Collection/CollectionSection.tsx [hunk 3]
```diff
@@ -58,6 +61,9 @@ const CollectionSection: React.SFC<IProps> = props => {
                                 <ArticleCard
                                     key={key}
                                     href={getArticleURL(resource)}
+                                    isLoggedIn={props.isLoggedIn}
```
[the remark this answers is not in the document]
Comment (gjeanmart) [reply app/containers/Collection/CollectionSection.tsx]: done

File: app/containers/Bookmark/EditBookmarkFolderWidget/index.tsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import View from './View'
+import { connect } from 'react-redux'
+import { openModalAction } from '../../../components/Modal/Module'
```
[the remark this answers is not in the document]
Comment (gjeanmart) [reply app/containers/Bookmark/EditBookmarkFolderWidget/index.tsx]: done

**Pull request merged**